lib/calendar_assistant/location_config_validator.rb

```diff
@@ -0,0 +1,12 @@
+# coding: utf-8
+class CalendarAssistant
+  class LocationConfigValidator
+    def self.valid?(config)
+      return if (config.calendar_ids - [ Config::DEFAULT_CALENDAR_ID ]).empty?
+      return if !!config[CalendarAssistant::Config::Keys::Settings::NICKNAME]
+      return if !!config[CalendarAssistant::Config::Keys::Options::FORCE]
+
+      raise BaseException, "Managing location across multiple calendars when a nickname is not set is not recommended, use --force to override"
```

Comment:
Do you think it's valuable to raise a specific subclass of BaseException? I'm torn, because I've been doing it, but I don't know that it actually makes a difference.

Reply:
I think we could do it, to set up the pattern for the future. We may want `ValidationException` to be handled differently eventually

Reply:
Raised #95